<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>489</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SHARP Autonomous Builder</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="toolBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="driveTab">
    <attribute name="title">
     <string>Drive</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="armTab">
    <attribute name="title">
     <string>Arm</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="elevatorTab">
    <attribute name="title">
     <string>Elevator</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="visionTab">
    <attribute name="title">
     <string>Vision</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="timeoutTab">
    <attribute name="title">
     <string>Timeout</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="startStopTab">
    <attribute name="title">
     <string>Start/Stop</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="generateButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load File</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="debug">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>104</width>
     <height>78</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Canvas</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
